web/src/components/logs-table.tsx

```diff
@@ -240,6 +250,87 @@ const LogRow: React.FC<LogRowProps> = ({ data, title, showResources }) => {
   return null;
 };
 
+const MetricsLink: React.FC<MetricsLinkProps> = ({ container, logType, namespace, pod }) => {
+  const { t } = useTranslation('plugin__logging-view-plugin');
+
+  const [isLoading, setIsLoading] = React.useState<boolean>(false);
+  const [error, setError] = React.useState<Error | undefined>();
+
+  const checkForMetrics = React.useCallback(() => {
+    setError(undefined);
+    const queryLabels: string[] = [];
+    if (container) {
+      queryLabels.push(`kubernetes_container_name="${container}"`);
+    }
+    if (namespace) {
+      queryLabels.push(`kubernetes_namespace_name="${namespace}"`);
+    }
+    if (pod) {
+      queryLabels.push(`kubernetes_pod_name="${pod}"`);
+    }
+
+    // We require a log type and at least one search label value to get related metrics
+    if (logType && queryLabels.length) {
```

Comment:
If this condition is not satisfied, then `checkForMetrics` will do nothing, but there will be no UI feedback to explain why. In other words, I think that clicking the button will do nothing, but there will be nothing to explain why. Maybe the button should not even be shown if this condition is true?